set.mm

```diff
@@ -424,6 +425,8 @@ useful for discussing objects (beyond simply true or false).  In particular,
   wla $f wff la $.
   $( Let variable ` ka ` be a wff. $)
   wka $f wff ka $.
+  $( Let variable ` al ` be a wff. $)
+  walpha $f wff al $.
```

Comment:
I remember at some point I proposed to move all such declarations at the beginning of the file, like here, but I was warned about a performance cost for the provers (albeit very small). It's not needed in the main section. Maybe it should be declared at the beginning of your mathbox, like `bnjwphm` and following, and `wjph` and following, for instance.

Reply:
Done